index.html

```diff
@@ -1,62 +1,221 @@
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400;500&display=swap" rel="stylesheet">
-</head>
-<body class="bg-img">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body class="bg-img">
     <header class="header">
-        <nav>
-            <button class="hamburger"><img src="./assets/icons/ic_menu.svg" alt="menu"></button>
-        </nav>
+      <nav>
+        <button class="hamburger">
+          <img src="./assets/icons/ic_menu.svg" alt="menu" />
+        </button>
+      </nav>
     </header>
     <main>
-        <section class="hero-section">
-            <div class="container">
-                <h1 class="heading">Nurgul Kereikhan</h1>
-                <p class="paragraph">
-                Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
-                </p>
-                <button class="cta-btn">Start collaboration</button>
+      <section class="hero-section">
+        <div class="container">
+          <h1 class="primary-heading">Nurgul Kereikhan</h1>
+          <p class="paragraph">
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Taka a look of my works. If you like what you
+            see and have a project you need coded, don’t hesitate and contact
+            me.
+          </p>
+          <button class="cta-btn">Start collaboration</button>
+        </div>
+        <img
+          class="decoration"
+          src="./assets/icons/image_geometry_2.svg"
+          alt="geometry2"
+        />
+      </section>
+      <section class="section project-section">
+        <div class="container">
+          <h2 class="primary-heading">Projects</h2>
+        </div>
+        <ul class="grid">
+            <li class="item">
+            <img
+              class="project-img"
+              src="./assets/images/Rectangle21.svg"
+              alt="Project-1"
+            />
+            <div class="container container-information">
+              <h2 class="heading">Project name goes here</h2>
+              <ul class="features">
+                <li class="feature">HTML/CSS</li>
+                <li class="feature">Ruby on Rails</li>
+                <li class="feature">JavaScript</li>
+              </ul>
+              <button class="c-projects">
+                See this project
+                <img src="./assets/images/Union.svg" alt="see this project" />
+              </button>
             </div>
```

Comment:
- [ ] I think it would be better If you align the content of your projects using flexbox because this will give you flexibility in terms of positioning your project content based on screen sizes and also for your website responsiveness 😄. 

Kindly use flexbox on the `item` class 👍🏽